user-interface/src/components/uswds/modal/Modal.tsx

```diff
@@ -39,6 +41,9 @@ function ModalComponent(props: ModalProps, ref: React.Ref<ModalRefType>) {
     if (!props.forceAction) {
       if (e.key === 'Escape') {
         close(e);
+      } else if (e.key === 'Tab') {
+        //tabbed to button to accept or close modal when tabbed all the way through modal
+        return;
```

Comment:
This can probably be removed, since we are now handling the tabs in the assignAttorneyModal rather than here.